storage/spanstore/metrics/decorator.go

```diff
@@ -66,9 +65,14 @@ func NewReadMetricsDecorator(spanReader spanstore.Reader, metricsFactory metrics
 }
 
 func buildQueryMetrics(namespace string, metricsFactory metrics.Factory) *queryMetrics {
-	qMetrics := &queryMetrics{}
 	scoped := metricsFactory.Namespace(namespace, nil)
-	metrics.Init(qMetrics, scoped, nil)
+	qMetrics := &queryMetrics{
+		Errors:     scoped.Counter("requests", map[string]string{"result": "err"}),
```

Comment:
Is there a better name than `requests` and/or `responses`?
`requests` = number of query operations performed
`responses` = number of items returned per request

Reply:
> responses = number of items returned per request

what items? traces, spans?

Reply:
It depends on which operation:  find traces, get operations, get services and get trace.
So `jaeger_find_traces_responses` would be tracking the number of traces returned.

Reply:
On reflection, `requests` and `responses` is probably ok - the `requests` is a counter, so should be obvious relates to number of requests. `responses` is a histogram to represent the number of items in each response.

Reply:
Seems this discussion has finished already, but how about "operations" and "results"? 

Even though "requests/responses" is probably OK as well, I would avoid it as it's a bit of a loaded term.

Reply:
Have removed the additional part of the name for those counters, so now it is just:
```
jaeger_query_find_traces{result="err"} 0
jaeger_query_find_traces{result="ok"} 1
jaeger_query_find_traces_latency_bucket{result="err",le="0.005"} 0
jaeger_query_find_traces_latency_bucket{result="ok",le="0.005"} 1
jaeger_query_find_traces_responses_bucket{le="0.005"} 1
```